src/fu-util.c

```diff
@@ -3107,15 +3106,16 @@ main (int argc, char *argv[])
 		return EXIT_FAILURE;
 	}
 	if (fwupd_client_get_tainted (priv->client)) {
-		g_autofree gchar *fmt = NULL;
-		/* TRANSLATORS: this is a prefix on the console */
-		fmt = fu_util_term_format (_("WARNING:"), FU_UTIL_TERM_COLOR_RED);
-		g_printerr ("%s %s\n",
-			    fmt,
-			    /* TRANSLATORS: the user is SOL for support... */
-			    _("The daemon has loaded 3rd party code and "
-			      "is no longer supported by the upstream developers!"));
-	}
+		/* TRANSLATORS: the user is SOL for support... */
+		msg =  _("The daemon has loaded 3rd party code and "
+			"is no longer supported by the upstream developers!");
+		fu_util_show_daemon_warning (msg);
+	}
+#ifndef SUPPORTED_BUILD
+	/* TRANSLATORS: the user is SOL for support... */
+	msg = _("This is not a distribution validated package, it is only best effort support");
```

Comment:
Maybe `Use of this 3rd party package is not supported by the upstream developers.`?

Reply:
I really don't want to tell someone just because they used the snap or hand built they're getting ZERO help.  Is there somewhere in the middle?

Reply:
`Use of this 3rd party package is not really supported by the upstream developers` or `Use of this 3rd party package is not fully supported by the upstream developers`?

Reply:
Hmm, I'm still not there with either of these messages.  If someone has gone through the trouble to successfully build 1.5.1, I would rather help them than an old Ubuntu 1.3.11.

Similarly with the snap, I can trust they got all the deps right because they're all in the snap already.

Reply:
How about `This is not a distribution validated package, it may not work properly with other fwupd based clients`

Reply:
Or even 
> This package is not validated by `$NAME`, it may not work properly with other fwupd based clients

Where `$NAME` comes from `/etc/os-release`.

Reply:
> This package is not validated by $NAME

I'm not sure that's true for Ubuntu; politically they want to push snaps everywhere. It's us who care. :(

Reply:
OK so how about just:
`This package has not been validated, it may not work properly with other fwupd based clients`

Reply:
Maybe even shorter, `This package has not been validated, it may not work properly.`